libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -572,6 +572,10 @@ float SRV_Channels::get_output_norm(SRV_Channel::Aux_servo_function_t function)
  */
 void SRV_Channels::limit_slew_rate(SRV_Channel::Aux_servo_function_t function, float slew_rate, float dt)
 {
+    if (slew_rate <= 0) {
```

Comment:
is_negative would allow anything within epsilon of 0 to be passed through to the check :) (The alternate would just be if (is_positive()) around all the existing code :)